features/src/test/scala/com/salesforce/op/features/types/FeatureTypeSparkConverterTest.scala

```diff
@@ -85,7 +85,7 @@ class FeatureTypeSparkConverterTest
   }
   property("create a feature type instance and back in a timely fashion") {
     forAllConcurrentCheck[FeatureTypeSparkConverter[_ <: FeatureType]](
-      numThreads = 10, numInvocationsPerThread = 10000, atMost = 10.seconds,
+      numThreads = 10, numInvocationsPerThread = 10000, atMost = 15.seconds,
```

Comment:
why the change?

Reply:
Seems to be that when parallel threads are scheduled for running tests, wait time on the CPU increased a bit. Tested this on my fork with travis and have seen this test taking ~13 sec on few occasions.